igcollect/logfile_values.py

```diff
@@ -23,7 +23,7 @@
 import os
 import gzip
 import logging
-import datetime
+from datetime import datetime, timezone
```

Comment:
Arguable, as you like that shadow the `datetime` root module. With the `datetime` import.

Reply:
`Line too long` I would not accept as reason. Stating, that we will not likely add anything else and the script no disadvantage about shadowing it, might be one.
Making just aware of it as it might be dangerous and can be still ok to do.